ChangeLog

```diff
@@ -1,5 +1,17 @@
 Version TBD (In Progress on the Develop Branch):
 
+Version 2020.10.20:
```

Comment:
```suggestion
Version 2021.03.02:
```

Reply:
Dam, I though I had updated that ....

Reply:
Fixed in https://github.com/saltstack/salt-bootstrap/pull/1542